docs/index.html

```diff
@@ -15,20 +15,6 @@
         <div class='py1 px2'>
           <h3 class='mb0 no-anchor'>react-d3-graph</h3>
           <div class='mb1'><code>2.0.0-rc2</code></div>
-          <h3>Quick Links</h3>
```

Comment:
Please discard the changes to this file, this should only be modified on a release commit.